packages/ncurses/build.sh

```diff
@@ -115,16 +122,3 @@ termux_step_post_make_install() {
 	tic -x -o $TI $TERMUX_PKG_SRCDIR/kitty-${TERMUX_PKG_VERSION[3]}/terminfo/kitty.terminfo
 	tic -x -o $TI $TERMUX_PKG_SRCDIR/alacritty-${TERMUX_PKG_VERSION[4]}/extra/alacritty.info
 }
-
-termux_step_post_massage() {
-	# Some packages want these:
-	cd "$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/include" || exit 1
-	rm -Rf ncurses{,w}
-	mkdir ncurses{,w}
-
-	local _file
-	for _file in *.h; do
```

Comment:
It would be better to check whether some symlinks are broken in `post_massage` step

Reply:
Good idea, will add it